packages/js/onboarding/src/images/payment-methods/wechat-pay.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * External dependencies
+ */
+import { createElement } from '@wordpress/element';
+
+export default () => (
+	<svg
+		width="64"
+		height="40"
+		viewBox="0 0 64 40"
+		fill="none"
+		xmlns="http://www.w3.org/2000/svg"
+	>
+		<rect width="64" height="40" fill="white" />
+		<path
+			className="cls-1"
```

Comment:
```suggestion
```

Reply:
Removed in https://github.com/woocommerce/woocommerce/pull/57436/commits/66b7bcb2dca370ecca8bafcdd9c0e34bcce3054d